fabric-api-lookup-api-v1/src/main/java/net/fabricmc/fabric/api/lookup/v1/block/BlockApiCache.java

```diff
@@ -40,25 +40,30 @@
 @ApiStatus.NonExtendable
 public interface BlockApiCache<A, C> {
 	/**
-	 * Retrieve an Api from a block in the world, using the world and the position passed at creation time.
+	 * Attempt to retrieve an API from a block in the world, using the world and the position passed at creation time.
 	 *
-	 * <p>Note: If the block state is known, it is more efficient to use {@link BlockApiCache#get(BlockState, Object)}.
+	 * <p>Note: If the block state is known, it is more efficient to use {@link BlockApiCache#find(BlockState, Object)}.
+	 *
+	 * @param context Additional context for the query, defined by type parameter C.
+	 * @return The retrieved API, or {@code null} if no API was found.
```

Comment:
```suggestion
	 * @param context additional context for the query
	 * @return the retrieved API, or {@code null} if no API was found
```
Imo you should move the description of type `C` to the `@param` tags for class doc

Reply:
I find `context additional context for the query` much harder to parse than `context Additional context for the query, defined by type parameter C.` as it's harder to directly tell where the parameter name ends and where the description starts.

Reply:
Also, the whole PR is consistent with itself and it's an entirely new module - nobody complained the first time and I really don't feel like going through all of the very extensive javadoc one more time to make it less readable in the end.

Reply:
You should just document what `C` is in class's javadoc. You have the C as parameter type, so people already know it's C

Reply:
In this particular case, the parameters are not documented in `BlockApiCache` because they are the same as those of `BlockApiLookup`, and the cache already refers to it. I added this comment in `BlockApiCache` to be consistent with `BlockApiLookup#find` where it's imo necessary anyway do to `find` being one of the API entrypoints, and people might miss the `@param <T>` at the end of the huge javadoc.

Reply:
> might miss the `@param <C>` at the end of the huge javadoc

All parameters are generated alike as they are block tags whose contents are appended to end of member output. Per your argument, you shouldn't add param tags for method parameters because they might be missed.

Imo it's still best to document them in class. If you document the params, javadoc will generate the link for param `C` so when you click the clickable `C` in method parameter types, you will be redirected to the definition of the type parameter at the top of the class

Reply:
IntelliJ does not seem to link the class directly, so I kept that comment in the javadoc of the `find` methods, but I also added the params to `BlockApiCache` for extra clarity and for web browser javadoc users.